src/main/java/com/coveros/selenified/Browser.java

```diff
@@ -136,11 +137,23 @@ public String getDetails() {
         if (getVersion() != null) {
             stringBuilder.append(" ").append(getVersion());
         }
-        if (getPlatform() != null) {
-            stringBuilder.append(" ").append(getPlatform());
+        if (getScreensize() != null || Property.runHeadless()) {
+            stringBuilder.append(" (");
+            if (getScreensize() != null ) {
+                stringBuilder.append(getScreensize().substring(0, 1).toUpperCase() + getScreensize().substring(1)).append(" ");
+            }
+            if(Property.runHeadless() ) {
+                stringBuilder.append("Headless ");
+            }
+            stringBuilder.setLength(stringBuilder.length() - 1);
+            stringBuilder.append(")");
         }
-        if (getScreensize() != null) {
-            stringBuilder.append(" ").append(getScreensize());
+        if (getPlatform() != null) {
+            String platform = getPlatform().getPartOfOsName()[0];
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "platform" which hides the field declared at line 85. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AHiddenFieldCheck)